must.go

```diff
@@ -103,44 +153,64 @@ func (b *Browser) MustWaitDownload() func() []byte {
 		info := wait()
 		path := filepath.Join(tmpDir, info.GUID)
 		defer func() { _ = os.Remove(path) }()
-		b, err := ioutil.ReadFile(path)
-		utils.E(err)
-		return b
+		data, err := ioutil.ReadFile(path)
+		b.must(err)
+		return data
 	}
 }
 
 // MustFind is similar to Browser.Find
 func (ps Pages) MustFind(selector string) *Page {
 	p, err := ps.Find(selector)
-	utils.E(err)
+	if err == nil && p == nil {
+		// When there is no match ps.Find returns nil, nil.
+		err = &ErrElementNotFound{}
+	}
+	if len(ps) > 0 {
+		ps[0].browser.must(err)
+	}
+	if err != nil {
+		// fallback to utils.E when we don't have the browser context.
+		utils.E(err)
+	}
 	return p
 }
 
 // MustFindByURL is similar to Page.FindByURL
 func (ps Pages) MustFindByURL(regex string) *Page {
 	p, err := ps.FindByURL(regex)
-	utils.E(err)
+	if err == nil && p == nil {
+		// When there is no match ps.Find returns nil, nil.
+		err = &ErrElementNotFound{}
+	}
+	if len(ps) > 0 {
+		ps[0].browser.must(err)
+	}
+	if err != nil {
+		// fallback to utils.E when we don't have the browser context.
+		utils.E(err)
+	}
 	return p
 }
 
 // MustInfo is similar to Page.Info
 func (p *Page) MustInfo() *proto.TargetTargetInfo {
 	info, err := p.Info()
-	utils.E(err)
+	p.browser.must(err)
```

Comment:
This one will cause nil pointer issue, if `p.Info()` fails, the info will be nil. The code below will panic:

```
fmt.Println(page.MustInfo().URL)
```